### PR TITLE
thuang-download-buttons-feedback

### PR DESCRIPTION
**Pull request by tihuan**
I updated the column to use `flex-grow` instead of percentage, since flex-grow is easier to use and less brittle, as shown below in OLD, where the percentage is consistent between the header and the column below, but somehow it's still not aligned

NEW:
Screen Shot 2020-09-15 at 12 51 47 PM

OLD:
Screen Shot 2020-09-15 at 12 51 47 PM

### CODE REVIEW

**Review by Bento007 — APPROVED** [review]
Simple, and easier to use.

**Comment by tihuan** [comment]
Thanks for the quick review, @Bento007 ! I'll wait for @pdugan20 's review before merging, since he's the CSS guru 😆 ! Thank you!

**Comment by tihuan** [comment]
Pat and Colin both approved in Slack! Merging into your branch now, @Bento007 🤩 🙏 Thank you!!

**Comment by pdugan20** [comment]
Looks great man!

**Pull request merged**
